packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewerContainer.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import asyncStates from '@zooniverse/async-states'
+import PropTypes from 'prop-types'
+import { withStores } from '@helpers'
+
+import locationValidator from '../../helpers/locationValidator'
+import useSubjectImage from '../SingleImageViewer/hooks/useSubjectImage'
+import FlipbookViewer from './FlipbookViewer'
+
+function storeMapper(store) {
+  const { frame: defaultFrame } = store.subjectViewer
+
+  return {
+    defaultFrame
+  }
+}
+
+function FlipbookViewerContainer({
+  defaultFrame = 0,
+  loadingState = asyncStates.initialized,
+  onError = () => true,
+  onReady = () => true,
+  subject
+}) {
+  /** This initializes an image element from the subject's defaultFrame src url.
+   * We do this so the SVGPanZoom has dimensions of the subject image.
+   * We're assuming all frames in one subject have the same dimensions. */
+  const defaultFrameUrl = subject ? Object.values(subject.locations[defaultFrame])[0] : null
```

Comment:
```suggestion
  const defaultFrameUrl = subject ? Object.values(subject.locations[defaultFrame - 1])[0] : null
```
Worth double-checking, but I think `defaultFrame` starts at 1, not 0. I've noticed on Etch-A-Cell, that the default frame is off by one when the subject loads. It should be the middle (third) frame but is the fourth in the new classifier.

https://frontend.preview.zooniverse.org/projects/h-spiers/etch-a-cell/classify/workflow/8921

Also, we've been using that project to test the CSSI work, but the subject viewer is broken now. 😞